examples/python/FullScriptSupport_Pandas/README.md

```diff
@@ -0,0 +1,45 @@
+# Example: Full script support using Pandas
+This example plugin includes support for all script functionality and is based on the original [Full Script Support](../FullScriptSupport/README.md) Python example. The implementation of this plugin differentiates mainly by the use of the Pandas library, the data received from Qlik is now saved to a Pandas data frame. Another difference is that we now use the `exec` method to evaluate the script rather than the `eval` method, as we did in the original example plugin. This change opens up the possibility to for instance pass a multiline script from Qlik.
+
+## Content
+* [Implementation](#implementation)
+    * [Parameters sent from Qlik](#parameters-sent-from-qlik)
+    * [TableDescription](#tabledescription)
+    * [Result](#result)
+* [Qlik Documents](#qlik-documents)
+* [Run the Example!](#run-the-example)
+
+## Implementation
+We aim for a well documented code for you to easily be able to follow along. If something is unclear, please let us know so that we can update and improve our documentation. In this file we will guide you through a few specific key points in the implementation, that is worth clarifying.
```

Comment:
We have tried to provide well documented code that you can easily follow along with. If something is unclear, please let us know so that we can update and improve our documentation. In this file, we guide you through a few key points in the implementation that are worth clarifying.